prusti-viper/src/encoder/mir_encoder.rs

```diff
@@ -104,12 +105,15 @@ impl<'p, 'v: 'p, 'r: 'v, 'a: 'r, 'tcx: 'a> MirEncoder<'p, 'v, 'r, 'a, 'tcx> {
     /// - `vir::Expr`: the place of the projection;
     /// - `ty::Ty<'tcx>`: the type of the place;
     /// - `Option<usize>`: optionally, the variant of the enum.
-    fn encode_projection(
+    pub fn encode_projection(
         &self,
         place_projection: &mir::PlaceProjection<'tcx>,
+        encoded_place: Option<(vir::Expr, ty::Ty<'tcx>, Option<usize>)>
```

Comment:
There is no change, is there? Currently, it just describes the return values, which remain unchanged.

Reply:
I'm referring to the new parameter on line 111. Could you add a note in the rustdoc explaining how the `encoded_place` parameter should be used? For example, it seems that `encoded_place` can be used to replace the encoding of `place_projection.base`, and this seems an important thing to mention. Also, what about renaming the parameter to `encoded_base_place`? It might be easier to understand that it's just the encoding of the base of the projection.

Reply:
Alright, it should be changed now.